projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/path-actions/class-simple-delete.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Optimizations\Page_Cache\Pre_WordPress\Path_Actions;
+
+use Automattic\Jetpack_Boost\Modules\Optimizations\Page_Cache\Pre_WordPress\Filesystem_Utils;
+use Automattic\Jetpack_Boost\Modules\Optimizations\Page_Cache\Pre_WordPress\Logger;
+use SplFileInfo;
+
+class Simple_Delete implements Path_Action {
+	/**
+	 * Delete a file or directory.
+	 *
+	 * @param SplFileInfo $file The file or directory to delete.
+	 * @return int The number of files or directories deleted.
```

Comment:
This mentions returning `int`, but `delete_file` returns a boolean.

Reply:
Good catch, I'll make apply_to_path return 1 after deleting.